windows-lb/network.tf

```diff
@@ -1,32 +1,55 @@
-
+## Copyright © 2021, Oracle and/or its affiliates. 
+## All rights reserved. The Universal Permissive License (UPL), Version 1.0 as shown at http://oss.oracle.com/licenses/upl
 
 /* Network */
 resource "oci_core_vcn" "vcn1" {
-  cidr_block     = "10.1.0.0/16"
+  cidr_block     = var.vcn_cidr
   compartment_id = var.compartment_ocid
   display_name   = "vcn1"
   dns_label      = "vcn1"
+  defined_tags   = { "${oci_identity_tag_namespace.ArchitectureCenterTagNamespace.name}.${oci_identity_tag.ArchitectureCenterTag.name}" = var.release }
 }
 
 resource "oci_core_subnet" "subnet1" {
-  cidr_block          = var.subnet_cidr[0]
-  display_name        = "subnet1"
-  dns_label           = "subnet1"
-  security_list_ids   = [oci_core_security_list.securitylist1.id]
-  compartment_id      = var.compartment_ocid
-  vcn_id              = oci_core_vcn.vcn1.id
-  route_table_id      = oci_core_route_table.routetable1.id
-  dhcp_options_id     = oci_core_vcn.vcn1.default_dhcp_options_id
+  cidr_block        = var.subnet1_cidr
+  display_name      = "subnet1"
+  dns_label         = "subnet1"
+  security_list_ids = [oci_core_security_list.securitylist1.id]
+  compartment_id    = var.compartment_ocid
+  vcn_id            = oci_core_vcn.vcn1.id
+  route_table_id    = oci_core_route_table.routetable1.id
+  dhcp_options_id   = oci_core_vcn.vcn1.default_dhcp_options_id
 
   provisioner "local-exec" {
     command = "sleep 5"
   }
+  defined_tags = { "${oci_identity_tag_namespace.ArchitectureCenterTagNamespace.name}.${oci_identity_tag.ArchitectureCenterTag.name}" = var.release }
 }
 
+resource "oci_core_subnet" "subnet2" {
```

Comment:
Nitpick : can we rename the subnets as compute subnet and dB subnet or similar . Seems more descriptive that way

Reply:
@jeevanjoseph it is a good catch. We can change it. On the other hand, the instance name should be rather connected to ASPNet.